### PR TITLE
fix: emitting input click events for datepicker

### PR DESCRIPTION
**Pull request by lloydaf**
**What:**

​This PR enables emitting the click event on input fields inside the `Datepicker` component, for both (single and range) variants.

**Why:**

​This is to keep in line with expected behavior, as datepicker components consist of an input field or two, depending on the variant, and so must detect the click.
**How:**

​Remove the `event.stopPropagation()` method call from the `Input` component `onClick` attribute
**Media:**

https://www.loom.com/share/29d22680ca7e4db0a9d16036996a7f71
**Checklist:**

- [x] Ready to be merged

### CODE REVIEW

**Review by nlopin — APPROVED** [review]

**Review by div-Leo — APPROVED** [review]

**Review by agonzalezUX — APPROVED** [review]

**Comment by duranmla** [comment]
@lloydaf we won't merge until you mark the checkbox of "ready to merge", it is there something missing here?

**Review by duranmla — APPROVED** [review]
Please check if this is ready for merge so we can move forward!

**Comment by lloydaf** [comment]
my bad, wasn't really sure who was supposed to mark that lmao. It's ready to merge 😬

**Pull request merged**
